debian/upstream/signing-key.asc

```diff
@@ -0,0 +1,97 @@
+jaN1Hsl1xke/ZiG4e1XxRnO615dOQdHE1k+KgEVXe/j/udp3l9PYm3MDkkVZc4dx
+FW0Vp8acWXf5r3WpESWvoTdL5Y6l7WVzowOzyuHRJIUX83n7HvaWBh4cLs4iYwd8
+Jl/ozo9mJ5WmJi/wdIDTLBAoQ9XLU06NYEF8d93XK+fT6NYDIdnPSZUx/zzv++K2
+PTfEnIwjvmI7hfwm34PhLaucHbQs5X7x2USO1/xRNFNCAh0uAHDoGuVm37lKe+Nu
+WvMZ/UuhgdzZImP+iQEzBBABCAAdFiEEk2wFAZFfZu/iUyvwDYiwhI/dSJsFAlld
+pLAACgkQDYiwhI/dSJupXgf+L6uwQpxvMYUsIxoTRrMhSE1woGsyLNKxr4GbKLyl
+ZsraCXtJ5XFtkZB0Nc+ypfADNNJlkl6njcqedHhe7EP0OJQh/zQjpBkP6hteWvpL
+MpwSB5HeClcnaFg1vt6YpKmJqX+wWvnlXI8IZrs2ML1Ex0mSbQCWKTv8rkm1lGEU
+Rd+1UGX6IK+xOX5yPIPHfFRf/f2pEvwTf8+4FUGe4JVYK1fWpgcU6D6KTLvgNWHr
+dbnU9L0c5Z+y2nZagfGC2EhE0uDF+CFXbFq4DY3oJ1aI1LGmVBNvV6c04Xa2AK39
+Tzyht2jhT2Q7XrOzakKV3plEekD5H2xvaEG4GFBLf+P3ObkCDQRW/s3cARAAvFxy
+XgCmnDADi0JzEp0LQehLlH8RuncAUaisgUItEyhrPIsf3l8eCjuyWwrvvTp93+gR
+seFvVBcrc/AKCvqJwrioyTrpWwwfaR8cgTBZspLSF+RwGU4KCWOD6vgNoM88/e31
+o7J51hwJ67Pg25xx2sOZC0Fc9lDsT095y/tgLC1BhKnfjX2pnpVH58NYhr9FPnCY
+44fO8/xlHkn2GO3XPNf+9uz1Nh5foV+vhMCp51IcsSy0c/AlXgXUgzu5Z1mIqrag
+FPfq/gtquqmxVpEjF6mMZZgpgzyA8GYXPdgswIrWLq35GJD+LNB+F8EO0E9om2e/
+Nj+zqOjJQLdvNl6IDQnqNSgwXV8kGX7GFvGc8nVkRVLqzISJ9GSFpN9c4QR/U9lV
+Q8ehN5/MucE8ymZFM8d3xUNGaXUEx5WABTg4SGNW4vvlCIWn/yJawfqg3A4biGb3
+Mqric4hyiwGL+uclJKB7978qh5wEdgObZCd6D8gJUPBXy+vdXhNa9Ou7v/MlcPVc
+zCdsHe6ChfIKgvuvgTXLLC798SKnBHkUQS4EDStv3NJu4OugyCk9Qc8iOmX3YW8Y
+mdTS71is97tvHjWEv9gAmyHMQs8rYLS87Vz78tA9/3JZoGpaosN7nWMScbXj0oxG
+tg9tdzhncPAfwsomMFvg4OfzJWB2PpYl6AI4DdsAEQEAAYkCJQQYAQoADwUCVv7N
+3AIbDAUJAeEzgAAKCRCJoq08B9li6HAXEACVkUNpFDRrZqCaxrmwAkwN3p0+65xg
+NyNSLbMrL93vi1PcgvLgo+1ZBfothwhNWnZ70grzH9QnVjowNIbKIYObzXDdkqiT
+Fp0+z/mJKhiJcDgwc7s8nNBgrBom/kH/hDk0QEM7XQAcLqeR8jgq8tIhN2cxurtH
+eeZArh1st+xIzBR3Q6ua2A0puzezAfFQR2LvRtLUNLGNvxajTur7S/D2NK0UrUUa
+CDuSRqSEH4moce9BlszjXA56D/wlCqrcv93cq908T/emgLNTEAXP26yUrKMGV1kf
+FQZZHLIpv61lSMJaTgD51gLJAgySc1y02Rn3j87PjXig7/U/QstR8N/UUmwUK3rr
+9KHCydiotKeivjJLNMJfUaOb+brv/fzpupk2zn5F+koUnqLAKPm9J+paRdFXrRv+
+eezk5zOHGglbcwDgbJn9hMVzKnVH6czoY+l4uSDRfv1IL20o8ajHATzOhPJMLMrs
+zaWYmqYLbIc9IGKPwU5js9L+Kxsky18aAjEB6sDIMFJDKbEhYrgD9dASxDDdMJFX
+mvDnM8ip+QBpfmpwnUH0IKcINYsVihRm/vynCTy2P9L+K4xnVi1pTBLvJPXjYcdy
+axT49RPEXE1iSG7+iXgG51lNTG/EaVfumsQhCdI6LB9z61fATMOQBJAbpVnWot4s
+nWqRr3lKXn384okCJQQYAQoADwIbDAUCWOJfAgUJA8TEpgAKCRCJoq08B9li6AVc
+EACVstNEmPS/NsucbUvEeFLxpOBlRrwMWb3L1TbGKjT5TZTfN/HhlLno6zkNllIa
+bjEiNaneTCgF/0ofRM+cMFYbGEkI/gmHwoXw4kka3fdK74+CAX5/o7sYbS1Y+mlT
+1hgrywh/yMq+PgGXQxl8VZqaNUK8geyAbDaGSsOw6QU+ktljtsfrLJhiwE81RhDT
+PBJJ0rwN6ST+M8Rl+7KsGpu1fm73D9v9WzFAnIIwEBoo8ON0VfJQ5NXAcSwf/qGN
+tQp+9rEuDENUtYhN9nMv6/ibO7lw8PM5TCzHhGQ2sp0Eob07hvXbgN72WibkcVj1
+z0rwixZnDCRP8qUGqVGlbaND2U84Usf2U7yLJpETbduwku/yoiDfyL/kdz8kEA4y
+U/dBBwjo34NtCHhTEiyrEJyPYzf+Hho1KSROZzjtCQleoYXiL89Dv96+4WWJ//4W
+/lXYCl/ljW0Py7XQ0IdZbhxeJht08VMebbrxiXBBWLWuS+/xrhBX3ajk002Xr1Ms
+SoIFHjORXUIGUpyHAT6sY6MlpIZP/76tY5yzAk8DqJ60MNbS+Jyrqk9eaZ+lc9uv
+t6V0KQe7Uhsd6pllGw6q5cBfyzyK3l8goADYT9ox3b+cRXgrYgHH9cTyG6Uf+rnz
+pvNV0uPx4mvSq/w6D1tWqWHLgBa4AgDxv3f8JxOqDrc/Ug==
+=p3C2
+-----END PGP PUBLIC KEY BLOCK-----
```

Comment:
I saw that this file is used together with a [`debian/watch` file and the `pgpsigurlmangle` option](https://www.debian.org/doc/debian-policy/#optional-upstream-source-location-debian-watch). Do we want to add such a file?

Reply:
I don't think it's mandatory. Do you think it's worth adding given our time constraints?

Reply:
We can add this later.